deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -87,3 +66,281 @@ rules:
+  - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - apiregistration.k8s.io
+  resources:
+  - apiservices
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authentication.k8s.io
+  resources:
+  - tokenreviews
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authorization.k8s.io
+  resources:
+  - selfsubjectaccessreviews
+  - subjectaccessreviews
+  - selfsubjectrulesreviews
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - authorization.openshift.io
+  resources:
+  - clusterrolebindings
+  - subjectaccessreviews
+  - resourceaccessreviews
+  - clusterroles
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - autoscaling.openshift.io
+  resources:
+  - clusterautoscalers
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view oauths, clusterversions, infrastructures, apiservers, consoles,
+#    authentications, builds, clusteroperators, dnses, schedulers, projects,
+#    featuregates, images, ingresses, networks, operatorhubs, proxies
+#    https://issues.redhat.com/browse/OSD-4298
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view consolelinks, consolenotifications, consoleexternalloglinks,
+#    consoleyamlsamples
+- apiGroups:
+  - console.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view flowschemas and prioritylevelconfigurations
+- apiGroups:
+  - flowcontrol.apiserver.k8s.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - image.openshift.io
+  resources:
+  - images
+  - imagesignatures
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view configs and imagepruners
+- apiGroups:
+  - imageregistry.operator.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view machineconfigs, machineconfigpools, kubeletconfigs,
+#    controllerconfigs
+- apiGroups:
+  - machineconfiguration.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - metal3.io
+  resources:
+  - provisionings
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view storagestates and storageversionmigrations
+- apiGroups:
+  - migration.k8s.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - network.openshift.io
+  resources:
+  - clusternetworks
+  - hostsubnets
+  - netnamespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingressclasses
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - node.k8s.io
+  resources:
+  - runtimeclasses
+  verbs:
+  - get
+  - list
+  - watch
+# osd-readers can view oauthaccesstokens, oauthauthorizetokens, oauthclients,
+#    oauthclientauthorizations
+- apiGroups:
+  - oauth.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Sounds bad to give oauth{authorize,access}tokens, do we have a use case driving the inclusion of these?

Reply:
I was following the [google sheet](https://docs.google.com/spreadsheets/d/11Ldw9DmxP8nwvIF7oSMyum_NJNXK9wy9aB_D5CxmNJA/edit#gid=15405060) we used to review the roles. Now thinking about it, I agree that it is not a good idea to give access to this. I will remove it.

Reply:
done